typefall.d.ts

```diff
@@ -96,48 +96,48 @@ declare interface IVector {
  */
 declare function Angle(pitch: number, yaw:number, roll: number): IAngle;
 
-declare class IAngle {
-    public p: number;
-    public y: number;
-    public r: number;
-    public pitch: number;
-    public yaw: number;
-    public roll: number;
+declare interface IAngle {
+    p: number;
+    y: number;
+    r: number;
+    pitch: number;
+    yaw: number;
+    roll: number;
 
-    public constructor(pitch: number, yaw: number, roll: number);
+    constructor(pitch: number, yaw: number, roll: number);
```

Comment:
Why is there a constructor here?

Reply:
oh I totally missed that. also I don't think that one was my fault. Didn't you do the Angle one?

Reply:
I'll remove it.

Reply:
The refactor script wasn't setup to remove constructors from classes. Remove this line in your commit and I'll pull changes.

Reply:
Alright, done.